src/shogun/regression/LeastAngleRegression.cpp

```diff
@@ -127,25 +130,26 @@ bool CLeastAngleRegression::train_machine(CFeatures* data)
 	fill(m_is_active.begin(), m_is_active.end(), false);
 
 	SGVector<float64_t> y = ((CRegressionLabels*) m_labels)->get_labels();
+	Map<VectorXd> map_y(y.vector, y.size());
 	SGMatrix<float64_t> Xorig = feats->get_feature_matrix();
```

Comment:
any reason for example why LARS shouldn't be able to work on `SGMatrix<float32_t>` ?
but yeah this should be addressed in a consequent PR ;)